bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/config/config.xml

```diff
@@ -140,7 +153,7 @@
 			<advanced>true</advanced>
 		</parameter>
 
-		<parameter name="refreshInterval" type="integer" min="20" unit="s">
+		<parameter name="refreshInterval" type="integer" min="60" unit="s">
```

Comment:
What is the correct default value below (line 159)? I believe it's currently documented as 10 minutes?

Reply:
I might have mixed up `home` and `home-coach`.

Reply:
the default value is 600 but the user is able to change it (without any valuable effect by the way). Going below 60 will expose him to being rejected by the server.

Reply:
Resolving this comment, see https://github.com/openhab/openhab-addons/pull/16574#discussion_r1541980321 for keeping it in one place.